int32 Id
int32 BeamId
int32 OverallLength
int32 FirstBlockLength
int32 SecondaryBlockLength
int32 ThirdBlockLength
int32 StartDistance
int32 EndDistance
int32 FirstBlockDistance
int32 SecondBlockDistance
int32 ThirdBlockDistance
float32 FirstForkSpace
float32 SecondForkSpace
float32 PaletteMatchScore
float32 PalleteMinimalMatchScore
bool FirstBlockDetected
bool MiddleBlockDetected
bool LastBlockDetected
bool PaletteOk